src/http-middlewares/before/add-query-params.js

```diff
@@ -26,9 +12,7 @@ const addQueryParams = req => {
   if (hasQueryParams(req)) {
     const splitter = req.url.includes('?') ? '&' : '?';
 
-    if (req.removeMissingValuesFrom.params) {
-      pruneMissingQueryParams(req.params);
-    }
+    Object.entries(req.params).forEach(normalizeEmptyParamField(req));
```

Comment:
Toyed with using `reduce` to return a new object that could get piped to later operations. In the end, we're mutating request many other places, so stuck with that pattern.